CONTRIBUTING.md

```diff
@@ -23,3 +23,55 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Building Dagger
+
+Dagger is built with [`bazel`](https://bazel.build).
```

Comment:
This is pretty obvious from the rest of the documentation.

Reply:
Yes. It's a link for those who'd rather start with reading more about bazel.